python/lsst/ts/dimm/dimm_csc.py

```diff
@@ -259,6 +275,72 @@ async def begin_standby(self, id_data):
         await super().begin_standby(id_data)
         self.cmd_standby.ack_in_progress(id_data, timeout=60)
 
+    def convert_int(self, value):
+        """Convert a value to an int, or 0 in case the conversion fails.
+
+        Parameters
+        ----------
+        value: `str` or `int`
+            The value to convert.
+
+        Returns
+        -------
+        `int`
+            The value converted to an int, or 0 in case the conversion fails.
+
+        """
+        try:
+            return int(value)
+        except ValueError:
+            return 0
+
+    def convert_float(self, value):
```

Comment:
same comments here as for `convert_int`. :)

Reply:
Agreed.